photocollage/gtkgui.py

```diff
@@ -385,7 +393,7 @@ def set_settings(self, button):
             dialog.apply_opts(self.opts)
             dialog.destroy()
             if self.history:
-                self.render_preview()
+                self.regenerate_layout()
```

Comment:
Please regenerate layout only if the number of cols changed. Users don't want the layout to change if they just chose a different border color.

Reply:
@adrienverge, I see that it has not been decided for a long time. I can reopen this PR with line self.regenerate_layout() instead of self.render_preview(). Ok?

Reply:
@andreynovikov95 I'm not sure it's a good idea, as this PR would bring more complexity to the tool, which is not needed here.